index.html

```diff
@@ -54,33 +54,40 @@ <h1>Bonjour, Hi! I'm Frank</h1>
                 <h2>English Content</h2>
                 <div class="feed">
                     <h3><a href="https://www.frankysnotes.com" target="_blank">Franky's Notes <i class="bi bi-box-arrow-up-right"></i></a></h3>
+                    <p class="feed-description">My personal blog where I share insights, tutorials, and thoughts about cloud computing, development, and technology.</p>
```

Comment:
Don't forget AI in the covered topics. And there is my weekly post Reading Notes;  an habit I started a long time ago, where I share a list of all the articles, blog posts, and books that catch my interest during the week.